apps/backend/lib/graphql-bookshelf-bridge/fetcher.js

```diff
@@ -124,20 +124,35 @@ export default class Fetcher {
     return matches[0][0]
   }
 
-  _loadMany (relation, { method, loader, tap, fetchOpts }) {
-    return this.loaders.relations.load({relation, method})
-    .tap(tap)
-    .then(instances => {
-      // N.B. this caching doesn't take into account data added by withPivot
-      instances.forEach(x => loader.clear(x.id).prime(x.id, x))
-      return loader.loadMany(instances.map('id'))
-      .then(models => {
-        if (!fetchOpts.querySet) return models
-        const cleanOpts = omitBy(x => isNull(x) || isUndefined(x), fetchOpts)
-        const optsWithDefaults = Object.assign({offset: 0, first: 20}, cleanOpts)
-        return presentQuerySet(models, optsWithDefaults)
-      })
-    })
+  async _loadMany (relation, { method, loader, tap, fetchOpts }) {
+    const instances = await this.loaders.relations.load({ relation, method })
+
+    // N.B. this caching doesn't take into account data added by withPivot
+    instances.forEach(x => loader.clear(x.id).prime(x.id, x))
+
+    const models = await loader.loadMany(instances.map('id'))
+
+    if (!fetchOpts.querySet) return models
+
+    // NOTE: querySetId generation, which may prove useful for graphql cache reconciliation in the
```

Comment:
Adds the querySetId. As mentioned in PR description, I may remove this soon though harmless as it is.